auditbeat/module/audit/file/metricset_test.go

```diff
@@ -98,25 +98,20 @@ func TestEventReader(t *testing.T) {
 			t.Fatal(err)
 		}
 
-		if runtime.GOOS == "windows" {
-			updated := readTimeout(t, events)
-			assert.Equal(t, Updated.String(), updated.Action)
+		received := readMax(t, 3, events)
+		if len(received) == 0 {
```

Comment:
Correct. That's the flexibility this test case needs because sometimes an "updated" event isn't sent (and idk why).